extension/apmconfigextension/opamp_callbacks.go

```diff
@@ -134,9 +128,9 @@ func (rc *remoteConfigCallbacks) onMessage(ctx context.Context, conn types.Conne
 
 	if message.GetRemoteConfigStatus() != nil && message.GetRemoteConfigStatus().Status == protobufs.RemoteConfigStatuses_RemoteConfigStatuses_APPLIED && bytes.Equal(remoteConfig.ConfigHash, message.RemoteConfigStatus.GetLastRemoteConfigHash()) {
 		rc.logger.Info("Remote config applied", zap.String("hash", hex.EncodeToString(remoteConfig.ConfigHash)), agentUidField)
-		agent.lastConfigHash = message.GetRemoteConfigStatus().GetLastRemoteConfigHash()
+		agent.LastConfigHash = message.GetRemoteConfigStatus().GetLastRemoteConfigHash()
 		rc.agentState.Store(agentUid, agent)
-	} else if !bytes.Equal(agent.lastConfigHash, remoteConfig.ConfigHash) {
+	} else if !bytes.Equal(agent.LastConfigHash, remoteConfig.ConfigHash) {
```

Comment:
This bytes.Equal check could be dropped now, then right? This same check is now done in `fetcherAPMWatcher.RemoteConfig()`.